src/v2/Components/Onboarding/Steps/Budget.tsx

```diff
@@ -27,98 +47,75 @@ const OptionsContainer = styled.div`
   `};
 `
 
-interface State {
-  selection: number | null
+const budgetOptions = {
```

Comment:
For simple key-value stuff like this its healthy to use TypeScript's new [as const](https://blog.logrocket.com/const-assertions-are-the-killer-new-typescript-feature-b73451f35802/) feature.